src/Magnum/GL/Buffer.h

```diff
@@ -1171,10 +1186,15 @@ class MAGNUM_GL_EXPORT Buffer: public AbstractObject {
          * @overload
          * @m_since{2019,10}
          */
-        template<class T> Buffer& setSubData(GLintptr offset, std::initializer_list<T> data) {
+        template<class T> Buffer& setSubData(GLintptr offset, Containers::ArrayView<T> data) {
```

Comment:
Oh, forgot to mention (and that's why all builds fail now) -- everything that used to take `std::initializer_list<T>` needs to be `Containers::ArrayView<const T>` now, not `Containers::ArrayView<T>` (the `const` is implicit in the init list type).

Reply:
Oh, you're right, I missed that. Any preference for east const or west? I'm seeing an east const associated with a pointer in `AbstractTexture.h` but west consts in other places, don't know if that's the standard coding style or not 😅